packages/medusa/src/api/routes/admin/orders/__tests__/cancel-claim.js

```diff
@@ -0,0 +1,64 @@
+import { IdMap } from "medusa-test-utils"
+import { request } from "../../../../../helpers/test-request"
+import { ClaimServiceMock } from "../../../../../services/__mocks__/claim"
+
+describe("POST /admin/orders/:id/claims/:claim_id/cancel", () => {
```

Comment:
These tests ensure that the endpoint only allow you to cancel claims which are related to the given order id